subprojects/language-java/src/main/java/org/gradle/api/internal/tasks/compile/incremental/deps/ClassDependentsAccumulator.java

```diff
@@ -29,39 +29,39 @@
 
 public class ClassDependentsAccumulator {
```

Comment:
Yes, but that's such a minor issue that I don't think it's worth complicating the code for. The cost of analysis is in actually looking at the classes, not in aggregating the results. We could stop looking at classes as soon as there is a full rebuild cause. But even that is probably not important, because broken jars should be a rare exception.